### DISCUSSION TITLE
Publish  in NeurIPS Workshop?

### DISCUSSION THREAD

**Discussion by venkycr** [ds4dm/ml4co-competition#11] [Q&A]
Thank you for organizing the competition. Is there an accompanying workshop in Neurips 21? Would there be an opportunity to submit a publication in that workshop, as we work on the compeition?

**Comment by gasse** [comment]
Hi @venkycr ,

Thank you for asking ! Yes there will be publication opportunities for participants. There will not be an accompanying NeurIPS workshop per se, but there will be proceedings for the competition as well as a (virtual) poster session during NeurIPS. See our [announcement](https://github.com/ds4dm/ml4co-competition/discussions/15) for more details.

Best,
Maxime
